### PR TITLE
Publish to central

### PR DESCRIPTION
**Pull request by sadevelopment**
This has package renaming, replacement of TeamHubApp with GitLiveApp. NPM publishing to main registry

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ nbproject/
 target/
 .gradle/
 
+/node_modules
+/src/commonMain/kotlin/.DS_Store
```
Comment (nbransby) [inline .gitignore]: update to all .DS_Stores

File: build.gradle.kts [hunk 2]
```diff
@@ -86,10 +86,9 @@ tasks {
         dependsOn(copyPackageJson, copyJS, copySourceMap)
         workingDir("$buildDir/node_module")
         if(Os.isFamily(Os.FAMILY_WINDOWS)) {
```
Comment (nbransby) [inline build.gradle.kts]: can remove this if now

Comment (sadevelopment) [reply build.gradle.kts]: Do you mean just Os.isFamily?

Comment (sadevelopment) [reply build.gradle.kts]: Sorry misread it

Comment (nbransby) [reply build.gradle.kts]: yeah the only reason to make it OS specific was to run 2 command in one line which youi have removed now

**Pull request merged**